src/components/ContentWrapper.jsx

```diff
@@ -51,41 +50,32 @@ const ContentWrapper = (props) => {
 
   const experiment = useSelector((state) => state?.experiments[experimentId]);
   const experimentName = experimentData?.experimentName || experiment?.name;
-
+  const { RUNNING, SUCCEEDED, NEEDS_RERUN } = pipelineStatus;
   const {
-    loading: backendLoading,
     error: backendError,
     status: backendStatus,
   } = useSelector((state) => state.backendStatus[experimentId] ?? initialExperimentBackendStatus);
+
   const backendErrors = [pipelineStatus.FAILED, pipelineStatus.TIMED_OUT, pipelineStatus.ABORTED];
 
   const pipelineStatusKey = backendStatus.pipeline?.status;
-  const pipelineRunning = pipelineStatusKey === 'RUNNING';
+  const pipelineRunning = pipelineStatusKey === RUNNING;
   const pipelineRunningError = backendErrors.includes(pipelineStatusKey);
 
   const gem2sStatusKey = backendStatus.gem2s?.status;
-  const gem2sRunning = gem2sStatusKey === 'RUNNING';
+  const gem2sRunning = gem2sStatusKey === RUNNING;
   const gem2sRunningError = backendErrors.includes(gem2sStatusKey);
   const completedGem2sSteps = backendStatus.gem2s?.completedSteps;
 
   const changedQCFilters = useSelector(
     (state) => state.experimentSettings.processing.meta.changedQCFilters,
   );
 
-  // This is used to prevent a race condition where the page would start loading immediately
-  // when the backend status was previously loaded. In that case, `backendLoading` is `false`
-  // and would be set to true only in the `loadBackendStatus` action, the time between the
-  // two events would allow pages to load.
-  const [backendStatusRequested, setBackendStatusRequested] = useState(false);
```

Comment:
Why was this removed?

Reply:
these are changes introduced by Marcell from this PR -https://github.com/biomage-ltd/ui/pull/499

Reply:
still, do work out what it is for and why it is added. Maybe Marcell mistakenly removed it? We cannot be merging in things in the code that we don't understand, because we risk breaking something.